zavod/zavod/helpers/sanctions.py

```diff
@@ -30,4 +43,19 @@ def make_sanction(
         sanction.add("country", dataset.publisher.country)
     sanction.add("authority", dataset.publisher.name)
     sanction.add("sourceUrl", dataset.url)
+
+    if program is not None:
+        sanction.set("program", program)
+        program_id = context.lookup_value("sanction", program)
```

Comment:
Can we make it `sanction.program`? `sanction` on its own is not descriptive of what this does.